platformio/platformio.ini

```diff
@@ -25,6 +25,10 @@ lib_deps =
   adafruit/Adafruit Unified Sensor @ ^1.1.5
   bblanchon/ArduinoJson @ ^6.19.3
   zinggjm/GxEPD2 @ ^1.4.5
+  ; WiFiManager seems to have some platformio build integration issues
+  ; possibly related to release tag conventions.
+  ; https://github.com/tzapu/WiFiManager/issues/1453
+  https://github.com/tzapu/WiFiManager.git@2.0.16-rc.2+sha.3a303ee
```

Comment:
A few things to note here since I ran into a few small issues getting this to work.
- Note the version is a "...-rc.2".
- Some of the integration with platformio, I think was due to release tagging conventions in the libraries repo.  
- If you just add `WiFiManager` to the `lib_deps` it will install a really old 0.6.x version and have different build errors.
- Another issue I had is that you get build errors after just adding the library to `lib_deps` until you have a file that `#include <WiFiManager.h>.  I'm not sure if this a common issue or specific to this library.  

With that noted the library seemed to work well enough for me to use personally but, you'll need to decide if it is right for this project.

Reply:
> * Another issue I had is that you get build errors after just adding the library to `lib_deps` until you have a file that includes (E.G.`#include <WiFiManager.h>`).  I'm not sure if this a common issue or specific to this library.

I've gotten around this issue by using `lib_ldf_mode = deep` in my local platformio.ini file  [Docs](https://docs.platformio.org/en/latest/librarymanager/ldf.html#dependency-finder-mode), this would see to point to this particular issue being related to the library.  Since platformio has mechanisms for compensating for it, I'm not certain it is really a bug or just differences in build configurations.

Reply:
Functionally there is no reason to add this to the platformio.ini now that the other 
changes have added using that reference the WiFiManager. 

However I want to ask if I should still do it.  There is one scenario, I can think of, where this would be good to do.  If for some reason somebody wants to do some testing without WiFiManager and removes the using they will encounter unexpected build errors and may not know what to do about it.

Reply:
Can you clarify what you mean? I want to make sure Im not misunderstanding. Are you talking about whether or not to include lib_ldf_mode = deep?

Reply:
Correct.

Reply:
If you can build from clean without it, then maybe I wouldn't include it. Sounds like you have played around with it and understand it better than I do, so I'll support whatever you decide.